src/styles/blocks/details.scss

```diff
@@ -0,0 +1,49 @@
+.details {
+  &__title {
+    @include sectionTitle;
+  }
+
+  &__wrapper {
+    @include grid;
+    row-gap: 40px;
+
+    @include onTablet {
+      row-gap: 48px;
+    }
+  }
+
+  &__button {
+    font-size: 15px;
+    font-weight: 700;
+    text-decoration: none;
+
+    height: 56px;
+    background-color: #fff;
+    color: $secondary-text-color;
+    border: none;
+    border-radius: 40px;
+    transition: all 0.3s;
```

Comment:
Don't use all for transition, it affects in a bad way on the website performance
Set only needed properties, that will be changed